numpy/linalg/lapack_lite/lapack.f.patch

```diff
@@ -0,0 +1,56 @@
+@@ -57759,9 +57767,10 @@
+ *     Scan up each column tracking the last zero row seen.
+          ILACLR = 0
+          DO J = 1, N
++            I = M
+-            DO I = M, 1, -1
+-               IF( A(I, J).NE.ZERO ) EXIT
+-            END DO
++C           DO I = M, 1, -1
++C              IF( A(I, J).NE.ZERO ) EXIT
++C           END DO
```

Comment:
[There's a patch here](https://github.com/igraph/igraph/blob/0f41313882a5a491a1bfb0d26a52feee569734ce/tools/lapack/lapack.patch#L8-L19) that probably better fixes this, which I should probably copy here.